packages/cli/package.json

```diff
@@ -26,6 +26,7 @@
     "docs": "yarn oclif-dev readme --multi --dir=../docs/command-line-interface && yarn prettier ../docs/command-line-interface/*.md --write",
     "lint": "tslint -c tslint.json --project tsconfig.json",
     "prepack": "yarn run build && oclif-dev manifest && oclif-dev readme",
+    "tarball:": "npm shrinkwrap && yarn oclif-dev pack && rm npm-shrinkwrap.json",
```

Comment:
i had a really bad time with `npm shrinkwrap` in the past... and then came `yarn`.... so i'm a bit hesistant to use it here. Also... take in consideration that the moment you use `npm`; it won't read `yarn.lock` so you're building something different